src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/swift/SwiftPermanentStorageException.java

```diff
@@ -0,0 +1,12 @@
+package org.fogbowcloud.saps.engine.core.archiver.storage.swift;
+
+import org.fogbowcloud.saps.engine.core.archiver.storage.exceptions.PermanentStorageException;
+
+public class SwiftPermanentStorageException extends PermanentStorageException {
+
```

Comment:
didn't like this class. a little gain for a new abstraction

Reply:
Added fixme.